app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt

```diff
@@ -60,6 +62,11 @@ class ChatViewHolder(view: View, clickListener: MessageViewHolder.ClickListener,
                             receivedTick?.setImageResource(R.drawable.ic_clock)
                         }
 
+                        resendMessage?.setOnClickListener {
+                            val activity = context
+                            (activity as ChatActivity).setText(model.content)
```

Comment:
Don't cast

Reply:
@liveHarshit can u please mention why so. I have found that in many cases in this app casting have been used to call the functions. And if we don't use casting we cannot get a refernce to such function. Also its an Activity, so if we cast it, I don't think it will arise any error.

Reply:
Learn about Kotlin smart cast. And check PR https://github.com/fossasia/susi_android/pull/1838 for your reference.

Reply:
Check comment: https://github.com/fossasia/open-event-android/pull/722#issuecomment-447546425

Reply:
Check comment by @iamareebjamal to understand what is the issue: https://github.com/fossasia/open-event-android/pull/722#issuecomment-447546425

Reply:
updated. Thanks : )